### ISSUE TITLE
install-oacs.sh needs build_dir and ns_install_dir fixes

### ISSUE THREAD

**Issue by apiskors** [gustafn/install-ns#10]
The install-oacs.sh script does not respect the build_dir and ns_install_dir variables set on the command line.  Thus if I set non-default values for those when running install-ns.sh, install-oacs.sh breaks because it ignores the values I give for those variables.  The fix is easy, just copy the one-line setup code for each variable from install-ns.sh.

**Comment by gustafn** [comment]
yes, the command-line arguments (as in install-ns.sh) are not supported install-oacs.sh.... also because of the saved configuration values of the run of install-ns.sh, which are saved in "nsConfig.sh". 

I think, the right way is to support at least "ns_install_dir" for install-oacs.sh, since "ns_install_dir" is the location from where "nsConfig.sh" is picked up. ... but also other command line configuration variables can make sense for the power-user to override, what was saved in nsConfig.sh".

Will look at this when i touch this code the next time
